Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/types.go

```diff
@@ -2505,8 +2505,8 @@ type SecurityContextConstraints struct {
 	// Capabilities in this field maybe added at the pod author's discretion.
 	// You must not list a capability in both AllowedCapabilities and RequiredDropCapabilities.
 	AllowedCapabilities []Capability
-	// AllowHostDirVolumePlugin determines if the policy allow containers to use the HostDir volume plugin
-	AllowHostDirVolumePlugin bool
+	// Volumes is a white list of allowed volume plugins.
```

Comment:
I think it would be helpful to define the format of this format, since as far as I know the constants aren't rippled out to the user-doc for this.